src/smart-components/role/add-role-new/cost-resources.js

```diff
@@ -4,6 +4,7 @@ import { Select, SelectOption, SelectVariant, Grid, GridItem, Text, TextVariants
 import useFieldApi from '@data-driven-forms/react-form-renderer/dist/esm/use-field-api';
 import useFormApi from '@data-driven-forms/react-form-renderer/dist/esm/use-form-api';
 import { getResourceDefinitions, getResource } from '../../../redux/actions/cost-management-actions';
+import './add-role-wizard.scss';
```

Comment:
Do we need this import? I would assume it was imported before. We should avoid duplicate imports.